test/features/search.feature

```diff
@@ -2,6 +2,36 @@
 Feature: Set of scenarios to test functionality of search filters on the search page
 
   Background:
+    Given the following definition for 'case fields'
+      | ID               | Label              | FieldType         | FieldTypeParameter |
+      | TextField        | Text Field         | Text              |                    |
+      | TextAreaField    | Text Area          | TextArea          |                    |
+      | DateField        | Date Field         | Date              |                    |
+      | AddressField     | Address Field      | Address           |                    |
+      | PhoneField       | Phone Field        | PhoneUK           |                    |
+      | NumberField      | Number Field       | Number            |                    |
+      | YesNoField       | Yes or No Field    | YesOrNo           |                    |
+      | CollectionField  | Collection Field   | Collection        | Text               |
+      | MarritalStatus   | Fixed List         | FixedList         | marritalStatusEnum |
+      | MoneyField       | Money Field        | MoneyGBP          |                    |
+      | DocumentField    | Document Field     | Document          |                    |
+      | EmailField       | Email Field        | Email             |                    |
+      | MultiSelectField | Multi Select Field | MultiSelectList   | regionalCentreEnum |
+      | CaseHistory      | History            | CaseHistoryViewer |                    |
+    Given the following definition for 'case enums'
```

Comment:
what do you mean by  'case enums'?
shall we change this to:
 `Given the following definition for 'fixed lists'` ?

Reply:
👍